cspm/rn/prisma-cloud-code-security-release-information/features-introduced-in-code-security-2023/features-introduced-in-code-security-may-2023.adoc

```diff
@@ -0,0 +1,30 @@
+== Features Introduced in May 2023
+
+Learn about the new Code Security capabilities on Prisma™ Cloud Enterprise Edition (SaaS) in May 2023.
+
+The following new features or enhancements are available for Prisma Cloud Code Security. These capabilities help agile teams add security checks to their existing IaC (Infrastructure-as-Code) model and enforce security throughout the build lifecycle.
+
+* <<new-features>>
+
+
+[#new-features]
+=== New Features
+
+[cols="50%a,50%a"]
+|===
+|FEATURE
+|DESCRIPTION
+
+|*Code Security Dashboard*
+|Code Security dashboard (*Dashboard > Code Security*) enables you to analyze error trends across code integrations on Prisma Cloud. While emphasizing on the critical and high severity code errors, the dashboard also  gives you an understanding of the potentially vulnerable vectors in repositories, policy errors including secrets, non-compliant package licenses, CVE vulnerabilities and IaC misconfigurations. See the historical trend for Code Issues and Pull Requests to make informed decisions to prevent the recurrence of the trend.
```

Comment:
```suggestion
|Code Security dashboard (*Dashboard > Code Security*) enables you to analyze error trends across code integrations on Prisma Cloud. While emphasizing the critical and high severity code issues, the dashboard also gives you an understanding of the potentially vulnerable vectors in repositories, policy errors including secrets, non-compliant package licenses, vulnerabilities and IaC misconfigurations. See the historical trend for Code Issues and Pull Requests to make informed decisions to prevent the recurrence of the trend.
```